convert2rhel/pkghandler.py

```diff
@@ -524,3 +526,20 @@ def install_additional_rhel_kernel_pkgs(additional_pkgs):
 def is_rhel_kernel_installed():
     installed_rhel_kernels = get_installed_pkgs_by_fingerprint(system_info.fingerprints_rhel, name="kernel")
     return len(installed_rhel_kernels) > 0
+
+
+def clear_yum_versionlock():
+    loggerinst = logging.getLogger(__name__)
+
+    if os.path.isfile(_VERSIONLOCK_FILE_PATH) and os.path.getsize(_VERSIONLOCK_FILE_PATH) > 0:
+        # The file may exist and be empty => no version locks
+        VERSIONLOCK_FILE.backup()
+
+        loggerinst.warn("yum versionlock plugin is in use. It may cause the conversion to fail.")
+        loggerinst.info("\nIf you choose to continue, the package version locks will be cleared.")
```

Comment:
I wasn't too keen on this sentence as it was using passive voice when asking the user for confirmation. I was struggling to come up with an active voice that worked, but Zack Hawkins helped and came up with this

```suggestion
		loggerinst.info("Do you want to continue? This will clear package version locks.")
```